docs/about/whats_new.rst

```diff
@@ -8,30 +8,65 @@ What's New
 Next Version
 ============
 
-- docs: fix broken links :pull:`1725`
-- Update dependencies :pull:`1727`, :pull:`1731`
+v1.9.3 (15th April 2025)
+========================
+
+Contains a fix to a serious bug affecting dask usage, preparation for psycopg3, and a lot of other minor cleanup,
+tweaks and fixes.
+
+Includes contributions from @pjonsson, @omad, @caitlinadams, @SpacemanPaul and first PRs from new contributors
+@emmanuel-ferdman and @allrob23.
+
+Special thanks to @pjonsson who is single-handedly responsible for most of the PRs in this release, and to
+to all supporting organisations including Geoscience Australia, CSIRO and RI.SE.
+
+Bug Fix
+-------
+
+- Fix error when run with Python 3.10 causing pickling errors :pull:`1776`
+
+CI Fixes and Improvements:
+--------------------------
+
+- CI: fix docker image name :pull:`1730`
+- CI: fix codecov warning :pull:`1740`
+- CI: retry more in setup-miniconda :pull:`1753`
+- CI: cancel old PR jobs :pull:`1761`
+- CI: update doctor rst version :pull:`1775`
+- Stop building Python 2 wheels :pull:`1752`
+- pytest: only test documentation for relevant files :pull:`1751`
+- Fix some readthedocs warnings :pull:`1762`
```

Comment:
I think this should go under documentation fixes or similar, it's at least potentially visible to the user.